modules/mesos-cluster/main.tf

```diff
@@ -76,7 +80,7 @@ resource "aws_launch_configuration" "mesos-master-ec2-lc" {
 
 resource "aws_launch_configuration" "mesos-slave-ec2-lc" {
   name_prefix                 = "mesos-slave-${var.environment}-lc-"
-  image_id                    = "${lookup(var.aws_amis, var.aws_region)}"
+  image_id                    = "${var.environment == "production" ? lookup(var.aws_amis, var.aws_region) : lookup(var.aws_amis, format("%s-%s", var.aws_region, "ubuntu-16-04"))}"
```

Comment:
Instead of ``lookup`` and ``format`` can't we hardcode 2 different AMIs as ``stage`` and ``prod``?